enabler/src/components/payment-methods/card/card.ts

```diff
@@ -64,20 +64,20 @@ export class Card extends BaseComponent {
 
       // Mock Validation
       let isAuthorized = this.isCreditCardAllowed(requestData.paymentMethod.cardNumber);
-
       const resultCode = isAuthorized ? PaymentOutcome.AUTHORIZED : PaymentOutcome.REJECTED;
 
-      if (resultCode === PaymentOutcome.AUTHORIZED) {
```

Comment:
If this line is removed, it will still send request to processor when credit card is not allowed. Is it expected?